lib/rspec/expectations/expectation_target.rb

```diff
@@ -73,7 +73,7 @@ def to(matcher=nil, message=nil, &block)
         # @param [String or Proc] message optional message to display when the expectation fails
         # @return [Boolean] false if the negative expectation succeeds (else raises)
         # @see RSpec::Matchers
-        def not_to(matcher=nil, message=nil, &block)
+        def not_to(matcher, message=nil, &block)
```

Comment:
Is there a reason for this? Or is this just desired? I'm pondering the ramifications

Reply:
Good question!
The reason was to allow this:
```ruby
is_expected.to > 3
```
and print a meaningful error `The expect syntax does not support operator matchers, so you must pass a matcher to #to.`
With the removal of `=nil`, it now is `wrong number of arguments (given 0, expected 1..2)` now instead.

I relied on our test suite and this change is a mistake.

Let me quickly revert this and add a spec for this message.

Reply:
Fixed, thanks for noticing!